cypress/support/fragments/data_import/mapping_profiles/newMappingProfile.js

```diff
@@ -74,5 +74,49 @@ export default {
       }
     }
     cy.do(Button('Save as profile & Close').click());
+  },
+
+  fillMappingProfileForUpdate:(specialMappingProfile = defaultMappingProfile) => {
+    cy.do([
+      TextField({ name:'profile.name' }).fillIn(specialMappingProfile.name),
+      Select({ name:'profile.incomingRecordType' }).choose(marcBib),
+      Select({ name:'profile.existingRecordType' }).choose(specialMappingProfile.typeValue)
+    ]);
+    if (specialMappingProfile.typeValue === folioRecordTypeValue.instance) {
```

Comment:
Try to think about cover this code into function and paste it inside your obejct. May be it better way may be not. Just think about this :) I mean I see here a lot of "if"
Example what do I mean:

```
const obj1 = {
  typeValue = 'something_1',
  function doSomething() {
    cy.do(...)
  }
}

const obj2 = {
  typeValue = 'something_2',
  function doSomething() {
    cy.do(...)
  }
}

fillMappingProfileForUpdate:(specialMappingProfile = defaultMappingProfile) => {
  cy.do([
      TextField({ name:'profile.name' }).fillIn(specialMappingProfile.name),
      Select({ name:'profile.incomingRecordType' }).choose(marcBib),
      Select({ name:'profile.existingRecordType' }).choose(specialMappingProfile.typeValue)
    ]);
    specialMappingProfile.doSomething()
}
```
In this way you avoid a lot of ifs. You just call function from object. May you don't need typeValue anymore.